src/main/java/school/lemon/changerequest/java/introduction/hw2/Task7.java

```diff
@@ -1,7 +1,19 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
+
+
 public class Task7 {
     public static int countEvens(int[] arr) {
-        return 0;
+        int count =0;
+        if(arr==null){return 0;}
+        for(int i=0; i<arr.length; i++){
+            if (arr[i]%2==0||arr[i]==0){count=count+1;}
```

Comment:
1. You can use increment. 
2. there is no need to check `arr[i]==0` because `0%2 == 0`